README.md

```diff
@@ -34,6 +34,10 @@ To get a local copy up and running follow these simple example steps.
 - Rails
 - Postgresql
 
+### Testing
+
+Run rspec/spec/folder_name/file_name
+
```

Comment:
- [ ] Please it will be nice to update your `Testing section` with `explicit steps` to clone and run your app tests locally, as this will be of help to none technical users who would like to know if your tests are passing. Kindly add the correct commands to run the tests.